### ISSUE TITLE
fails with pyclipper 1.1.0.post

### ISSUE THREAD

**Issue by typemytype** [typemytype/booleanOperations#50]
seems like it fails with the current version of pyclipper 

```traceback
"/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/booleanOperations/booleanOperationManager.py", line 70, in _performOperation
    operation, subjectFillType="nonZero", clipFillType="nonZero")
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/booleanOperations/booleanOperationManager.py", line 57, in clipExecute
    raise ExecutionError(exc)
booleanOperations.exceptions.ExecutionError: Execution of clipper did not succeed!
```

**Comment by anthrotype** [comment]
I suppose this is to do with the zero-area paths, following 
https://github.com/typemytype/booleanOperations/pull/49

**Comment by typemytype** [comment]
had some time to look at it: this traceback is raised when there are no contours given... like in a space or a glyph with only components.

**Comment by typemytype** [comment]
we urgently needs test cases:

```python
from booleanOperations.booleanOperationManager import clipExecute


testSubjectContours = [
    [[(100, 100), (200, 200), (200, 100)]],
    [[(100, 100), (200, 200)]],
    [[]],
    []
]


for testContour in testSubjectContours:
    try:
        r = clipExecute(testContour, [], "union")
        print("succes:", testContour)
    except Exception as e:
        print("failed:", testContour)
        print(e)
```

**Comment by typemytype** [comment]
it should not run `clipExecute` whenever the subjectContour is and empty list, even if its a a empty list of empty segments

**Comment by anthrotype** [comment]
Fixed by #51
